sgt-back-end/remove.js

```diff
@@ -0,0 +1,41 @@
+const db = require('./db');
+const { jsonError, sendStatus } = require('./status');
+const errorCheck = require('./errorCheck');
+
+function remove(app) {
+  app.delete('/api/grades/:gradeId', (req, res, next) => {
+
+    const sql = `
+      DELETE
+          FROM "grades"
+        WHERE "gradeId" = $1
+      RETURNING *
+    ;
+  `;
+
+    const gradeId = Number(req.params.gradeId);
+    const params = [gradeId];
+    const course = null;
+    const score = null;
+    const name = null;
+
+    errorCheck(gradeId, name ?? 'not required', course ?? 'not required', score ?? 'not required', req, res);
```

Comment:
It seems to me that `errorCheck` isn't doing much for you here. All you need to check is the gradeId. But that's fine, it makes the code symmetrical. However, you can just pass 'not required' instead of using `??` since you know those are not defined.